docker-compose.yml

```diff
@@ -103,30 +104,38 @@ services:
       - FORMS_SERVICE_PUBLIC_HOST=http://localhost:3009
       - FORMS_SERVICE_PRIVATE_HOST=http://form-runner:3009
       - AUTHENTICATOR_HOST=http://localhost:3004
+      - RSA256_PUBLIC_KEY_BASE64=LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUlHZk1BMEdDU3FHU0liM0RRRUJBUVVBQTRHTkFEQ0JpUUtCZ1FDb2xHc1dBYlc5VlBuNGZvVEVvL3V3WUV6QQpQSUNCRFlERzZmVFpwbXVVK0pyVWZmQXo5dGhpMUc0MktDREFBbGxTMkpkRWlFZTA2c2l6cnlEald5YkJ2MnF2CkpMS2k0WVZ0SXhvdGM3WUZ1K2YzSG1QUzJaOElTTHQrTFUwY29iSkFWTERKSHpIWVpPYnVtY1ExYWxHL08vN0wKUkd0VDVyU2VOQllQK2Q5Szl3SURBUUFCCi0tLS0tRU5EIFBVQkxJQyBLRVktLS0tLQ==
       - FLASK_ENV=development
     ports:
       - 3008:8080
   form-runner:
-    build: 
+    build:
       context: ../funding-service-design-frontend
-      args:
-        # To use branch of digital-form-builder change 'latest' to an alternative tag from https://github.com/communitiesuk/digital-form-builder/pkgs/container/digital-form-builder-dluhc-runner/versions and run 'docker compose build form-runner'
-        - BASE_IMAGE_TAG=latest
-    ports: 
+    ports:
       - 3009:3009
+    depends_on:
+      - form-runner-data
     environment:
+      - SINGLE_REDIS=true
+      - REDIS_HOST=form-runner-data
+      - REDIS_PORT=6379
+      - REDIS_PASSWORD=123abc
+      - 'VCAP_SERVICES={"redis":[{"credentials":{"host":"form-runner-data","port":6379,"password":"123abc"}}]}'
```

Comment:
Did we add this just to test the `VCAP_SERVICES` integration? Can we remove it as it confuses  how the binding works?

Reply:
Yep sure, removed